libgalois/test/dynamic-bitset.cpp

```diff
@@ -135,6 +135,106 @@ const Invariant NotValues =
 
 const std::vector<Invariant> invariants = {NotAndCount, NotValues};
 
+katana::Result<void>
+CountAndOffsetsTest() {
```

Comment:
FWIW, I think this test could be reframed as an invariant that is agnostic to its input (the way the the other tests in the file are structured).